src/mixing.html

```diff
@@ -298,7 +298,39 @@ <h4 id="mixing_intent_examples">Intent Examples</h4>
 <blockquote>x prime<br/> x superscript prime end superscript</blockquote>
 </div>
 
-    <p>Similarly an over bar may represent complex conjugation, or mean (average), again with possible readings with and without <code class="attribute">intent</code>:</p>
+<p>Language-near intent values can also be annotated compositionally, via the underscore function.</p>
```

Comment:
I'm not sure "Language-near intent values can also be annotated compositionally" is understandable without having been in multiple zoom calls previously. Space isn't at a premium, Perhaps spell it out more;

The intent `_` function, which does not generate speech, may be used to compose intent expressions which are close to  natural language.